hack/update-api-reference-docs.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Generates updated api-reference docs from the latest OpenAPI spec for
+# clusterregistry apiserver. The docs are generated at docs/api-reference
+# Usage: ./update-api-reference-docs.sh <absolute output path>
+
+set -exuo pipefail
```

Comment:
Remove the `x`

Reply:
Good catch! You mean you don't want all the debug output too?! :wink: